media/static-bundles.json

```diff
@@ -1940,6 +1947,14 @@
       ],
       "name": "mozilla-vpn-invite"
     },
+    {
+      "files": [
+        "protocol/js/protocol-details.js",
```

Comment:
Nit: I don't think you need to add `protocol-details.js` here as it's already in our common bundle?